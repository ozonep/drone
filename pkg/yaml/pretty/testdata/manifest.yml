---
kind: pipeline
name: default

clone:
  disable: true
  depth: 50

platform:
  os: linux
  arch: arm64
  variant: 7
  version: 1803

steps:
- name: build
  image: golang
  commands:
  - go get
  - go build

- name: test
  image: golang
  commands:
  - go test -v

trigger:
  status:
  - success
  branch:
  - master
  - develop

depends_on:
- foo
- bar

---
kind: secret
name: username
data: N2NmYjA3ODQwNTY1ODFlY2E5MGJmOWI1NDk0NDFhMTEK

---
kind: cron
name: nightly

spec:
  schedule: "1 * * * *"
  branch: master
  deployment:
    target: production

---
kind: signature
hmac: N2NmYjA3ODQwNTY1ODFlY2E5MGJmOWI1NDk0NDFhMTEK
