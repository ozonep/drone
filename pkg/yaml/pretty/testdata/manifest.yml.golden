---
kind: pipeline
name: default

platform:
  os: linux
  arch: arm64
  variant: 7
  version: 1803

clone:
  depth: 50
  disable: true

steps:
- name: build
  image: golang
  commands:
  - go get
  - go build

- name: test
  image: golang
  commands:
  - go test -v

trigger:
  branch:
  - master
  - develop
  status:
  - success

depends_on:
- foo
- bar

---
kind: secret
name: username
data: >
  N2NmYjA3ODQwNTY1ODFlY2E5MGJmOWI1NDk0NDFhMTEK

---
kind: cron
name: nightly
spec:
  schedule: 1 * * * *
  branch: master
  deployment:
    target: production

---
kind: signature
hmac: N2NmYjA3ODQwNTY1ODFlY2E5MGJmOWI1NDk0NDFhMTEK

...
